### ISSUE TITLE
Scrolling forward/backward near time 0 shows different results

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mcellteam/cellblender#23]
```
When stepping through visualization files, I've noticed the following odd 
behavior:

1: Show intro.cellbin.0000.dat, all molecules appear correct
2: Show intro.cellbin.0001.dat, all move, but some (all?) newly created 
molecules don't appear
3: Show intro.cellbin.0002.dat, all molecules appear correct again (all show up 
and move)
4: Show intro.cellbin.0001.dat (go back), all are now correct and the "missing" 
molecules appear
5: Show intro.cellbin.0000.dat (go back again), all appear as in Step 1.
6: Show intro.cellbin.0001.dat (go forward again), newly created molecules 
still don't appear.

To summarize, stepping forward seems to skip some of the molecules that are 
newly created after initialization (possibly only on the first step), but 
stepping backward appears to display them all properly.

I've given this a low priority, but I did want to document it.
```

Original issue reported on code.google.com by `cnlkucze...@gmail.com` on 9 Apr 2013 at 2:22
